execution_engine/src/core/runtime/mod.rs

```diff
@@ -2149,6 +2178,24 @@ where
 
         let target_key = Key::Account(target);
 
+        if let (false, Some(is_source_admin)) = (
+            self.config.allow_unrestricted_transfers(),
+            self.config
+                .is_account_administrator(&self.context.get_caller()),
+        ) {
+            let is_target_admin = self
+                .config
+                .is_account_administrator(&target)
+                .expect("is_account_administrator() returns Some(_) on a private chain");
```

Comment:
`fn is_account_administrator` returns Some(is_admin) if there's at least one admin configured, otherwise None - so if one call returns Some, all subsequent calls will also return Some